tests/unit/test_query.py

```diff
@@ -99,11 +101,19 @@ def test_execute_without_fetch(self):
 
     def test_add_query_success(self):
         cursor = mock.Mock()
-        with mock.patch.object(SQLConnectionManager, "add_query") as mock_add_query:
+        with (mock.patch.object(SQLConnectionManager, "add_query") as mock_add_query,):
```

Comment:
Is this right? It looks like maybe a typo.

Reply:
Artifact of prior code that was there. Cleaned up 🖖